zipline/assets/_assets.pyx

```diff
@@ -203,6 +203,26 @@ cdef class Equity(Asset):
         params = ', '.join(strings)
         return 'Equity(%d, %s)' % (self.sid, params)
 
+    property security_start_date:
+        """
+        DEPRECATION: This property should be deprecated and is only present for
```

Comment:
Are we intentionally choosing not to deprecate it now, or can we warn with a DeprecationWarning at runtime instead?  Or maybe this is a cython convention? (I don't know too much cython.)